src/main/packages/syntax-tree/syntax-tree-nonterminal/syntax-tree-nonterminal-component.tsx

```diff
@@ -17,6 +17,6 @@ export const SyntaxTreeNonterminalComponent: SFC<Props> = ({ element }) => (
   </g>
 );
 
-interface Props {
+export interface Props {
```

Comment:
why are you exporting all of these Props? I thought they are only required locally for the component